src/lib/server/twilio.ts

```diff
@@ -338,7 +338,7 @@ export async function sendNotif() {
 				await sendMsg({ phone, type: 'reminder' }, null);
 
 				// update reminder date for next notif -- x days from today
-				const newReminderDate = new Date(now);
+				const newReminderDate = new Date(reminderDatetime);
```

Comment:
It looks like you're loading as a UTC date but saving here in local time.

Reply:
you're right, sorry x.x now that we're not saving reminderDatetime in the user's local time, we can get rid of all timezone logic in the reminder func. pushed a fix~